src/bluesearch/embedding_models.py

```diff
@@ -247,7 +246,9 @@ def embed_many(self, preprocessed_sentences):
         ----------
         https://huggingface.co/transformers/model_doc/bert.html#transformers.BertModel
         """
-        return self.embed(preprocessed_sentences)
+        n_samples = len(preprocessed_sentences["input_ids"])
+
+        return self.embed(preprocessed_sentences).reshape(n_samples, -1)
```

Comment:
What was the motivation behind this change?

Reply:
The test is not passing without it:) If we have a single sample in the batch the `squeeze` inside of our source code is going to remove the sample dimension:)

Reply:
I see. Thanks for the explanation.